rustv1/test-utils/src/waiter.rs

```diff
@@ -86,3 +86,31 @@ impl WaiterBuilder {
         Waiter::new(self.max.unwrap_or(MAX_WAIT), self.poll.unwrap_or(POLL_TIME))
     }
 }
+
+#[macro_export]
+macro_rules! wait_on {
+    (
+        $req: expr,
+        $test: expr
+    ) => {
+        wait_on!($crate::waiter::Waiter::default(), $req, $test)
+    };
+    (
+        $waiter: expr,
```

Comment:
cool so that can configure the max wait settings? not super clear from the macro itself that that's what the waiter is for.

Reply:
Added comments